packages/unplugin-typegpu/test/aliasing.test.ts

```diff
@@ -14,13 +14,13 @@ describe('[BABEL] tgpu alias gathering', () => {
 
     expect(babelTransform(code)).toMatchInlineSnapshot(`
       "import hello from 'typegpu';
-      const increment = hello['~unstable'].fn([])(($ => (globalThis.__TYPEGPU_META__ ??= new WeakMap()).set($.f = () => {
+      const increment = (globalThis.__TYPEGPU_AUTONAME__ ?? (a => a))(hello['~unstable'].fn([])(($ => (globalThis.__TYPEGPU_META__ ??= new WeakMap()).set($.f = () => {
```

Comment:
maybe we should turn off plugin autonaming for the unplugin tests that aren't meant for testing this particular feature (turned off by default in tests and enabled in auto-naming.test.ts)? what do you think?